.changeset/calm-cougars-learn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-node': minor
+---
+
+Adapter Node: log getRequest error
```

Comment:
we prefix messages with `fix:` or `feat:`. the whole changelog is for adapter-node, so noting that again wouldn't add anything
```suggestion
fix: log error details when invalid request body is encountered
```